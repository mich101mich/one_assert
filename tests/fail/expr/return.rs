fn main() {
    one_assert::assert!(return);
    fn with_value() -> i32 {
        one_assert::assert!(return 1);
        0
    }
}
