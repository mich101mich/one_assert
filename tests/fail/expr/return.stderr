error: Expected a boolean expression, found a return statement
 --> tests/fail/expr/return.rs:2:25
  |
2 |     one_assert::assert!(return);
  |                         ^^^^^^

error: Expected a boolean expression, found a return statement
 --> tests/fail/expr/return.rs:4:29
  |
4 |         one_assert::assert!(return 1);
  |                             ^^^^^^^^
